ansible/mnesia_snapshot.yml

```diff
@@ -24,7 +24,7 @@
     - name: Archive Mnesia Directory
       archive:
         format: "{{ format }}"
-        path: "{{ project_root }}/{{ node_config.chain.db_path }}"
+        path: "{{ project_root + '/' if additional_storage is not defined else '' }}{{ node_config.chain.db_path }}"
```

Comment:
I see that [ticket has error](https://www.pivotaltracker.com/n/projects/2152594/stories/165713405) in upload to S3. Could you please link the failing job? I am struggling to follow the value of `node_config.chain.db_path`.

Reply:
Here is a failed job https://circleci.com/gh/aeternity/infrastructure/12074

Reply:
@lucafavatella configuration is loaded per environment name by https://github.com/aeternity/infrastructure/pull/371/files#diff-bd112c469bc1a7317e68d1a60294db6fL11

For example, for `main` environment in [node_config](https://github.com/aeternity/infrastructure/blob/master/ansible/vars/aeternity/main.yml#L49) `db_path` is set to [`/data/db{{ db_version|mandatory }}`](https://github.com/aeternity/infrastructure/blob/master/ansible/vars/aeternity/main.yml#L108)

Reply:
I see that the main.yml file has `additional_storage: true` so this change changes the behaviour in that CI run. Ok.

Reply:
I see (based on experiment on https://cryptic-cliffs-32040.herokuapp.com ) that the precedence of `if` and `+` is such that if the condition is false (i.e. if `additional_storage` is defined) then even `project_root` is skipped.

Reply:
Expected value of variables at this point on [CI](https://circleci.com/gh/aeternity/infrastructure/12074):
```
project_root: "{{ ansible_env.HOME }}/node"
additional_storage: true
node_config.chain.db_path: "/data/db{{ db_version|mandatory }}"
```